datafusion/core/src/physical_optimizer/update_aggr_exprs.rs

```diff
@@ -52,6 +52,7 @@ use datafusion_physical_plan::{
 pub struct OptimizeAggregateOrder {}
 
 impl OptimizeAggregateOrder {
+    #[allow(missing_docs)]
```

Comment:
Perhaps you could add some comments (even though I know it would be pretty boiler plate like "make new `OptimizeAggregateOrder`"